packages/wallet/src/services/HandlesTracker.ts

```diff
@@ -30,6 +30,12 @@ const handleInfoEquals = (a: HandleInfo, b: HandleInfo) =>
   deepEquals(a.tokenMetadata, b.tokenMetadata) &&
   deepEquals(a.nftMetadata, b.nftMetadata);
 
+const getHandleFromAssetId = (assetId: Cardano.AssetId) => {
+  const assetName = Cardano.AssetId.getAssetName(assetId);
+  const assetNameContent = Asset.AssetNameLabel.decode(assetName)?.content;
+  return Buffer.from(assetNameContent || assetName, 'hex').toString('utf8');
```

Comment:
Would be great to add a test for this

Reply:
https://github.com/input-output-hk/cardano-js-sdk/blob/5f412c32136f6b5ae9f1941bd6e9c7002daefdcb/packages/projection/src/operators/Mappers/withHandles.ts#L28

Maybe hoist this some code to `core` package as a util?

Reply:
Fixed here https://github.com/input-output-hk/cardano-js-sdk/pull/786/commits/92c602e8e9b9e2b1716bb1ef8e505c2da66b1dba.

Reply:
Let's put the new util under `AssetId` for consistency:
- Cardano.AssetId.getAssetName(assetId);
- Cardano.AssetId.getAssetNameAsText(assetId);

Reply:
Fixed here: https://github.com/input-output-hk/cardano-js-sdk/pull/786/commits/fff340d753da88419a10f077ac85a6e70ea9e58f and https://github.com/input-output-hk/cardano-js-sdk/pull/786/commits/0285146045d3bc7e7da971cc036cd90e43fe75dc